test/e2e/reconciler/apiexport/apiexport_controller_test.go

```diff
@@ -119,3 +123,70 @@ func TestRequeueWhenIdentitySecretAdded(t *testing.T) {
 		return gotHash == "e9cee71ab932fde863338d08be4de9dfe39ea049bdafb342ce659ec5450b69ae"
 	}, wait.ForeverTestTimeout, 100*time.Millisecond, "expected identity valid condition and matching hash, got hash %s", gotHash)
 }
+
+func TestVirtualWorkspacesAssigned(t *testing.T) {
+	t.Parallel()
+
+	server := framework.SharedKcpServer(t)
+
+	ctx, cancel := context.WithCancel(context.Background())
+	t.Cleanup(cancel)
+
+	orgClusterName := framework.NewOrganizationFixture(t, server)
+	workspaceClusterName := framework.NewWorkspaceFixture(t, server, orgClusterName)
+	t.Logf("Running test in cluster %s", workspaceClusterName)
```

Comment:
nit: do we *really* need the org workspace? why?
nit: the `NewWorkspaceFixture()` helper already logs this, no need to double